libcodechecker/generic_package_suppress_handler.py

```diff
@@ -48,7 +48,7 @@ def __revalidate_suppress_data(self):
     def store_suppress_bug_id(self, bug_id, file_name, comment):
 
         if self.suppress_file is None:
-            return False
+            return True
```

Comment:
The `__revalidate_suppress_data()` reads the suppression information from the file. This `if` statement is running when this file is missing.